sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Bugs Fixed
+- Removed `aiohttp` import from `azure.storage.blob.aio._download_async`.  #24965
```

Comment:
Super nit: See line 69 for an example of mentioning an issue in the changelog (just for consistency sake 😄)